pkg/sql/catalog/bootstrap/metadata.go

```diff
@@ -101,18 +101,12 @@ func (ms *MetadataSchema) AddDescriptor(desc catalog.Descriptor) {
 // AddDescriptorForSystemTenant is like AddDescriptor but only for the system
 // tenant.
 func (ms *MetadataSchema) AddDescriptorForSystemTenant(desc catalog.Descriptor) {
-	if !ms.codec.ForSystemTenant() {
-		return
-	}
 	ms.AddDescriptor(desc)
 }
```

Comment:
We may want to remove these functions and use AddDescriptor directly. With this change the comments are lying a bit.